.yamato/mobile-build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+2021.2_Build_Android_player:
+  name: 2021.2 Build Android player
+  agent:
+      type: Unity::VM
+      # Any generic image can be used, no need to have Android tools in the image for building
+      # All Android tools will be downloaded by unity-downloader-cli
+      image: package-ci/win10:stable
+      flavor: b1.small
+
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - mkdir C:\TMP
+    - cd C:\TMP && unity-downloader-cli -u 2021.2 -c editor -c Android -w --fast
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+    - utr.bat --suite=playmode --platform=Android --editor-location=C:\TMP\.Editor --testproject=testproject --player-save-path=build/players --artifacts_path=build/logs --scripting-backend=mono --build-only --testfilter=Unity.Netcode.RuntimeTests
```

Comment:
I would recommend using IL2CPP instead of mono since that is the most common backend for Android. I would also recommend passing in some sort of timeout for UTP we do `--timeout=1800` I would also pass in a recount setting here again for UTP we use `-reruncount=2` Also since this is a build only job I would recommend passing in `--extra-editor-arg=-batchmode --extra-editor-arg=-nographics'`